scripts/es2k/setup_infra.sh

```diff
@@ -117,9 +106,10 @@ function copy_certs() {
     rm -rf $STRATUM_DIR/certs/client.key
     rm -rf $STRATUM_DIR/certs/stratum.crt
     rm -rf $STRATUM_DIR/certs/stratum.key
-    cp $BASE_DIR/tls/certs/infrap4d/* /usr/share/stratum/es2k/certs/.
+    cp $BASE_DIR/scripts/tls/certs/infrap4d/* /usr/share/stratum/es2k/certs/.
+    # BUG:14020043045 workaround
```

Comment:
I don't think we should be putting internal bug numbers out in open source

Reply:
ok